addons/fire/config.cpp

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"ace_common"};
```

Comment:
because script_comonent does `#include "\z\ace\addons\medical_engine\script_macros_medical.hpp"`
```suggestion
        requiredAddons[] = {"ace_medical_engine"};
```